config/example.config.json

```diff
@@ -1,18 +1,21 @@
 {
   "development": {
-     "username": "root",
-    "password": null,
-    "database": "database_test",
+    
```

Comment:
there should only be 2 files changes. 
this and .gitignore

Reply:
.gitignore was updated int earlier commit, following is the commit number 
**aa70cd8**